lodash/index.d.ts

```diff
@@ -12335,7 +12335,7 @@ declare module _ {
          * @param value The value to check.
          * @return Returns true if value is null, else false.
          */
-        isNull(value?: any): boolean;
+        isNull(value?: any): value is null;
```

Comment:
Tests are failing since this is for the `types-2.0` branch and the tester expects the `master` branch. Just make the change and I'll look through the tests to make sure they are all OK.